transport/src/main/java/io/netty/channel/DefaultChannelPipeline.java

```diff
@@ -1118,6 +1142,20 @@ private void callHandlerCallbackLater(AbstractChannelHandlerContext ctx, boolean
         }
     }
 
+    private void wakeUpForAutoFlush() {
+        if (autoFlush && ENQUEUE_WAKEUP_TASK_UPDATER.compareAndSet(this, 0, 1)) {
+            if (null == wakeupTask) {
+                wakeupTask = new Runnable() {
+                    @Override
+                    public void run() {
+                        ENQUEUE_WAKEUP_TASK_UPDATER.set(DefaultChannelPipeline.this, 0);
+                    }
+                };
+            }
+            channel().eventLoop().execute(wakeupTask);
+        }
+    }
+
```

Comment:
@jchambers Yes those are the only steps required to use this feature. From the stack, it looks like you have  a very large number of connections or there is a bug :)

@normanmaurer This is the add failure for `tailTasks`, it would _not_ grow with writes, but will only grow with channels (1 task per channel) with pending flushes.